tv/tv.cpp

```diff
@@ -142,6 +148,7 @@ optional<smt::smt_initializer> smt_init;
 optional<llvm_util::initializer> llvm_util_init;
 TransformPrintOpts print_opts;
 unordered_map<string, FnInfo> fns;
+unordered_map<string, float> fns_elapsed_time;
```

Comment:
Declared a separate map here because if a function `f` has unsupported instructions `fns` may not have a slot for `f`.